### ISSUE TITLE
About the accuracy

### ISSUE THREAD

**Issue by Yuuuuuuuuuuuuuuuuuummy** [yuanyuanli85/Stacked_Hourglass_Network_Keras#24]
Hi yuanyuanli85,
I trained the network with my own dataset. I didn't test the network on MPII cause I see someone test it. I meet a confused problem, that's the accuracy during training is not stable, varies from 0.20 to 0.89 at most. I can't find the reason. I used Adam and lr is 5e-4. Maybe it is related to the learning rate? Hope to get your reply! Much Thanks!

**Comment by yuanyuanli85** [comment]
Could you post all your training log? so that i can take a look.

**Comment by Yuuuuuuuuuuuuuuuuuummy** [comment]
@yuanyuanli85 
![Screenshot from 2020-01-10 17-29-04](https://user-images.githubusercontent.com/34826192/72142069-e535a880-33ce-11ea-8ce1-a6ecbb6150bf.png)

Here is my screenshot of my log.The coloums which are labeld by yellow are my training acc and val acc. I merge the 4 stacks by average. The value of val acc is not stable.  BTW my dataset is about 1k images, I don't think it is overfitting. Really need your help ! thanks

**Comment by yuanyuanli85** [comment]
The val loss is not smooth.  If I understand correctly,  val loss 0.0015 -> val acc 0.148,  while val loss 0.005 -> val acc 0.22069.  Low val loss should lead to high val acc.  so, val loss/acc is not aligned with each other.  I think that is a good point to start to investigate.

**Comment by Yuuuuuuuuuuuuuuuuuummy** [comment]
I see. Thanks for your advice. I will try to solve this. Much Thanks!
